### ISSUE TITLE
Sonata Classification manager problem

### ISSUE THREAD

**Issue by milosmoto** [sonata-project/SonataMediaBundle#783]
Hi I have problem with adding sonata media bundle, after I add and configure , there is problem with some service.

Any help on this?

[Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]  
  The service "sonata.media.admin.media" has a dependency on a non-existent service "sonata.classification.manager.category".

**Comment by NeuralClone** [comment]
I had a similar issue when I first started using the bundle. If you're using the master branch like I am, it has a dependency on SonataClassificationBundle, which isn't mentioned in the documentation:

https://github.com/sonata-project/SonataMediaBundle/blob/master/composer.json

```json
"require": {
        "symfony/symfony": "~2.3",
        "sonata-project/core-bundle": "~2.2",
        "sonata-project/notification-bundle": "~2.2",
        "sonata-project/easy-extends-bundle": "~2.1",
        "sonata-project/doctrine-extensions": "~1.0",
        "sonata-project/classification-bundle": "~2.3",
```

You should be able to get it to work if you follow the instructions for installing the SonataClassificationBundle:

https://sonata-project.org/bundles/classification/master/doc/reference/installation.html

I'm not sure if the documentation for that bundle is 100% up to date but I don't recall having any problems getting it to work. It's pretty straightforward.

**Comment by Trip09** [comment]
other option is to remove the dependency, since this should be decouple from the classification bundle. I will work on this.

**Comment by NeuralClone** [comment]
Well, yes. That would be ideal. I was simply suggesting a way to get it working as is. :)

**Comment by x500** [comment]
@NeuralClone Thanks!

**Comment by tolstoyleo** [comment]
is there an easier way to remove the classification bundle? use an older version? i'm also facing this issue but classification bundle is not an option for us.

**Comment by Trip09** [comment]
@tolstoyleo is a PR to solve this issue... you will need to wait for https://github.com/sonata-project/SonataMediaBundle/pull/794

**Comment by Okimbellum** [comment]
Try to add
`new Sonata\ClassificationBundle\SonataClassificationBundle(),`
into AppKernel.php 
It works for me.

**Comment by k-alex** [comment]
@Okimbellum Many thanks 

this worked

Try to add
new Sonata\ClassificationBundle\SonataClassificationBundle(),
into AppKernel.php
It works for me.

**Comment by jordisala1991** [comment]
Fixed on latest release of MediaBundle 3.4.0. If you are using SonataMediaBundle it no longer require to have SonataClassification, and if is the other way around, this issue should be on SonataClassification.
